charmhelpers/contrib/hahelpers/apache.py

```diff
@@ -34,6 +34,8 @@
     INFO,
 )
 
+KEYSTONE_CA_CERT_FILE = 'keystone_juju_ca_cert'
```

Comment:
The goal of this PR is to stop making use of this filename, so should this be named `OLD_KEYSTONE_CA_CERT_FILE`? 
Also why is this used in `install_ca_cert()`. I'm sure it wasn't like that in earlier version of this PR but because you push-force (it's fine, it's your right :) ) I can't really show evidence of that. But this is basically the source of my confusion: I know see a PR that creates a file named `keystone_juju_ca_cert` although the PR description says it wants to move away from that name. What am I missing?

Reply:
basically I expect this PR to show me some variation of

```
install_ca_cert(<the new name>)
remove_cert(<the old name>)
```

but I'm confused because on line 88 of this file I see `install_ca_cert(<the old name>)`

Reply:
> The goal of this PR is to stop making use of this filename, so should this be named `OLD_KEYSTONE_CA_CERT_FILE`?
> Also why is this used in `install_ca_cert()`. I'm sure it wasn't like that in earlier version of this PR but because you push-force (it's fine, it's your right :) ) I can't really show evidence of that. But this is basically the source of my confusion: I now see a PR that creates a file named `keystone_juju_ca_cert` although the PR description says it wants to move away from that name. What am I missing?

The push-force was after a rebase, the diff is unchanged

Reply:
I have updated the commit message which I hope is now a but clearer.